.php-cs-fixer.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$finder = (new PhpCsFixer\Finder())
+    ->files()
+    ->in([__DIR__.'/app', __DIR__.'/tests', __DIR__.'/config', __DIR__.'/database', __DIR__.'/vendor/biigle'])
+    ->name('*.php')
+    ->ignoreDotFiles(true)
+    ->ignoreVCS(true);
+
+return (new PhpCsFixer\Config())
+    ->setRules([
+        '@PSR2' => true,
+        'ordered_imports' => true,
+        'single_space_after_construct' => true,
```

Comment:
This seems to be [deprecated](https://cs.symfony.com/doc/rules/language_construct/single_space_after_construct.html) with `single_space_around_construct` as replacement.